randomizer.py

```diff
@@ -1485,13 +1486,50 @@ def RandomizeDominion(setNames=None, options=None):
             counter += 1
 
         # Get final list of landscape cards
```

Comment:
This comment is no longer entirely accurate since it's not the final list yet. It would be more accurate to say "Determine list of landscape cards" or something along those lines